.typos.toml

```diff
@@ -0,0 +1,2 @@
+[default.extend-words]
+Algorythm = "Algorythm"
```

Comment:
with this https://crates.io/crates/typos-cli passes without error

happy to add it to the CI if you like

Reply:
I'm not sure about the configuration yet, but what is the difference to this?

```
[default.extend-identifiers]
TheAlgorythm = "TheAlgorythm"
```